discovery-provider/es-indexer/src/indexNames.ts

```diff
@@ -1,7 +1,7 @@
 export const indexNames = {
-  playlists: 'playlists9',
-  reposts: 'reposts9',
-  saves: 'saves9',
-  tracks: 'tracks9',
-  users: 'users9',
+  playlists: 'playlists10',
```

Comment:
@stereosteve double checking if this is right.. do we up the version any time there's a change to indexing?

Reply:
yeah since we're only changing `playlists` index we can just increment playlists and not the others...

playlists index takes a minute or two to build... but tracks / users can take 20 or 30 minutes... so nice to avoid that when possible.